modules/misc/news.nix

```diff
@@ -1333,13 +1333,23 @@ in
           A new module is available: 'wayland.windowManager.sway'
         '';
       }
+
       {
         time = "2020-03-04T18:55:03+00:00";
         condition = hostPlatform.isLinux;
         message = ''
           A new module is available: 'programs.abook'
         '';
       }
+
+      {
+        time = "2020-03-05T00:00:00+00:00";
+        condition = hostPlatform.isLinux;
+        message = ''
+          The NixOS integration module has a new option: 'home-manager.useGlobalPkgs'.
+          To learn more, see the installation section of the manual.
```

Comment:
Any chance of a small-ish description, other than "see the manual"?

Reply:
Yes, fixed.